codewof/static/svg/icons8-points.svg

```diff
@@ -0,0 +1 @@
.565 C 107.888 199.006,106.164 197.141,105.632 196.420 C 105.099 195.699,103.794 194.574,102.732 193.920 C 101.669 193.266,100.800 192.606,100.800 192.453 C 100.800 191.953,97.028 188.333,95.898 187.748 C 95.292 187.434,94.362 186.507,93.833 185.689 C 93.304 184.870,92.360 183.713,91.735 183.119 C 89.005 180.519,87.322 178.771,86.249 177.421 C 85.616 176.625,84.356 175.634,83.449 175.219 C 81.754 174.444,80.400 172.692,80.400 171.274 C 80.400 170.832,80.058 169.371,79.639 168.026 L 78.878 165.581 80.141 164.618 C 82.794 162.594,84.497 162.297,98.527 161.406 C 108.427 160.778,113.348 160.116,116.000 159.055 C 119.698 157.575,125.628 156.654,137.000 155.791 C 156.922 154.280,158.400 153.853,161.558 148.711 C 162.676 146.890,164.370 144.566,165.322 143.546 C 166.673 142.097,167.164 141.178,167.564 139.346 C 168.274 136.090,169.916 132.542,171.220 131.444 C 172.891 130.038,173.858 127.688,174.987 122.294 C 175.114 121.686,176.105 120.209,177.189 119.012 C 178.451 117.618,179.318 116.219,179.600 115.118 C 180.812 110.389,181.616 108.660,183.561 106.604 C 184.900 105.188,185.682 103.945,185.982 102.756 C 187.332 97.406,188.161 95.470,189.882 93.650 C 190.845 92.633,191.727 91.350,191.843 90.800 C 192.699 86.736,194.156 83.030,195.466 81.580 L 196.893 80.000 200.200 80.000 L 203.507 80.000 204.902 81.544 M26.544 99.969 C 25.113 100.842,25.012 100.978,24.639 102.536 C 24.386 103.593,24.505 103.533,26.383 101.655 C 29.007 99.031,29.064 98.432,26.544 99.969 M373.082 99.985 C 373.787 100.414,374.724 101.449,375.164 102.283 C 375.604 103.117,375.973 103.658,375.982 103.484 C 376.031 102.611,374.336 100.099,373.413 99.679 C 371.816 98.951,371.646 99.109,373.082 99.985 M26.975 102.500 L 25.800 103.800 27.100 102.625 C 28.312 101.530,28.584 101.200,28.275 101.200 C 28.206 101.200,27.621 101.785,26.975 102.500 M20.519 112.384 C 19.815 112.813,18.960 113.704,18.619 114.362 C 17.714 116.113,17.863 116.322,18.910 114.769 C 19.410 114.026,20.426 113.010,21.169 112.510 C 22.817 111.399,22.299 111.299,20.519 112.384 M378.000 111.835 C 378.000 112.110,382.347 116.400,382.626 116.400 C 382.731 116.400,382.445 115.673,381.992 114.784 C 381.096 113.028,378.000 110.741,378.000 111.835 M378.000 113.725 C 378.000 113.794,378.585 114.379,379.300 115.025 L 380.600 116.200 379.425 114.900 C 378.330 113.688,378.000 113.416,378.000 113.725 M389.314 129.720 C 389.801 130.226,390.274 130.587,390.364 130.523 C 390.598 130.355,389.142 128.800,388.752 128.800 C 388.574 128.800,388.827 129.214,389.314 129.720 M5.200 148.800 C 4.760 149.240,4.405 149.825,4.412 150.100 C 4.419 150.384,4.597 150.304,4.824 149.914 C 5.044 149.537,5.618 148.952,6.101 148.614 C 6.583 148.276,6.758 148.000,6.489 148.000 C 6.220 148.000,5.640 148.360,5.200 148.800 M393.400 148.378 C 394.289 148.742,395.378 149.881,395.716 150.800 C 395.888 151.266,395.944 151.240,395.968 150.682 C 396.009 149.756,394.250 147.992,393.318 148.025 C 392.686 148.048,392.696 148.090,393.400 148.378 M395.600 249.730 C 395.600 250.31
```

Comment:
There's no newline for EOF, but also the entirety of the file is on one line so I'm not sure if it's a problem